server/routes/prisonIncentiveLevels.ts

```diff
@@ -31,22 +34,36 @@ export default function routes(router: Router): Router {
       incentivesApi.getPrisonIncentiveLevels(prisonId),
     ])
 
-    const activeLevelCodes = new Set(prisonIncentiveLevels.map(prisonIncentiveLevel => prisonIncentiveLevel.levelCode))
```

Comment:
i would’ve preferred to use sets but they’re pretty pointless if you need to turn them back into arrays to get the difference between two sets